packages/vue-app/template/head.js

```diff
@@ -0,0 +1,29 @@
+<%= isTest ? '/* eslint-disable quotes, semi, indent, comma-spacing, key-spacing, object-curly-spacing, space-before-function-paren  */' : '' %>
+const head = <%= serializeFunction(head) %>
+<%= isTest ? '/* eslint-enable quotes, semi, indent, comma-spacing, key-spacing, object-curly-spacing, space-before-function-paren */' : '' %>
+
+<%
+// Inject link with alias
+if (head.link) {
+  head.link.forEach((link, index) => {
```

Comment:
I believe we prefer to use for..in or for..of in nuxt. Also the reference to link should be kept so you shouldnt need to use index to set the element within the array.